CefSharp/Callback/ICompletionCallback.cs

```diff
@@ -14,5 +16,10 @@ public interface ICompletionCallback : IDisposable
         /// Method that will be called once the task is complete. 
         /// </summary>
         void OnComplete();
+
+        /// <summary>
+        /// Returns a Task that was used to set the result of the CookieCallback 
+        /// </summary>
+        Task SetResultTask { get; }
```

Comment:
The Task had no place being part of the call back and needs to be removed from all instances

Reply:
Then how do I wait for the result to be set?

Reply:
How is that relevant to the interfaces?

Reply:
Removed.  Also pull in your change.